content/docs/lists-and-keys.md

```diff
@@ -6,25 +6,25 @@ prev: conditional-rendering.html
 next: forms.html
 ---
 
-First, let's review how you transform lists in JavaScript.
+पहले, आइए समीक्षा करें कि आप जावास्क्रिप्ट में लिस्ट्स को कैसे बदलते हैं।
 
-Given the code below, we use the [`map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) function to take an array of `numbers` and double their values. We assign the new array returned by `map()` to the variable `doubled` and log it:
+
+नीचे दिए गए कोड में हम `numbers` की एक array लेने और उनके मूल्यों को दोगुना करने के लिए मैप फ़ंक्शन का उपयोग करते हैं। हम मैप द्वारा दिए गए नए array को `double` नाम के वेरिएबल को सौपते हैं और फिर लॉग करते हैं।
 
 ```javascript{2}
 const numbers = [1, 2, 3, 4, 5];
 const doubled = numbers.map((number) => number * 2);
 console.log(doubled);
 ```
+यह कोड [2, 4, 6, 8, 10] को कंसोल पर लोग कर देगा।
 
-This code logs `[2, 4, 6, 8, 10]` to the console.
-
-In React, transforming arrays into lists of [elements](/docs/rendering-elements.html) is nearly identical.
+रियेक्ट में,array को लिस्ट ऑफ़ [एलिमेंट्स](/docs/rendering-elements.html) में बदलना लगभग समान है।
```

Comment:
`लिस्ट ऑफ़ [एलिमेंट्स](/docs/rendering-elements.html)` => `[एलिमेंट्स] की लिस्ट(/docs/rendering-elements.html)`

Reply:
`रियेक्ट` => `React`
we are not translating `React` word, for complete list of these words you can check #2

Reply:
`array को  [एलिमेंट्स] ` => `array को [एलिमेंट्स] `

remove extra space

Reply:
`[एलिमेंट्स] की लिस्ट(/docs/rendering-elements.html)` => `[एलिमेंट्स](/docs/rendering-elements.html) की लिस्ट`

Reply:
@anchal7299 this is not fixed yet

Reply:
@anchal7299 You need to change this
```
[एलिमेंट्स] की लिस्ट(/docs/rendering-elements.html)
```

to 

```
[एलिमेंट्स](/docs/rendering-elements.html) की लिस्ट
```

Reply:
` में,  array` => ` में, array`

remove extra space